src/styles/utils/variable.scss

```diff
@@ -0,0 +1,14 @@
+$cell-width: 100px;
+$cell-hight: 100px;
```

Comment:
There is a typo in the variable name `$cell-hight`. It should be `$cell-height` to maintain consistency and avoid errors in your stylesheets.